<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LSB隐写</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/icons/icons/icon.png</normaloff>:/icons/icons/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="readImageButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>读取图片</string>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/icons/icons/loadimage.png</normaloff>:/icons/icons/loadimage.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="embedButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>嵌入</string>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/icons/icons/embed.png</normaloff>:/icons/icons/embed.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="saveImageButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>存储图片</string>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/icons/icons/save.png</normaloff>:/icons/icons/save.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="extractButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>提取</string>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/icons/icons/extract.png</normaloff>:/icons/icons/extract.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/icons/icons/clear.png</normaloff>:/icons/icons/clear.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="originalImageLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>350</width>
      <height>250</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>原始图片</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="modifiedImageLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>350</width>
      <height>250</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>嵌入信息后的图片</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imageInfoLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>740</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>图片信息：宽度，高度，类型</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="maxLengthLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>740</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>最大可嵌入信息长度：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="messageTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>350</width>
      <height>80</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>要嵌入的信息</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="extractedMessageTextEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>420</y>
      <width>350</width>
      <height>80</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>提取出的信息</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="useEncryptionCheckBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>使用密钥</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="encryptionKeyLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>650</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>输入密钥</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
